src/Features/Core/Portable/ExtractMethod/MethodExtractor.Analyzer.SymbolMapBuilder.cs

```diff
@@ -1,87 +0,0 @@
-﻿// Licensed to the .NET Foundation under one or more agreements.
-// The .NET Foundation licenses this file to you under the MIT license.
-// See the LICENSE file in the project root for more information.
-
-#nullable disable
-
-using System.Collections.Generic;
-using System.Threading;
-using Microsoft.CodeAnalysis.LanguageService;
-using Microsoft.CodeAnalysis.Shared.Extensions;
-using Microsoft.CodeAnalysis.Text;
-using Roslyn.Utilities;
-
-namespace Microsoft.CodeAnalysis.ExtractMethod;
-
-internal abstract partial class AbstractExtractMethodService<
-    TStatementSyntax,
-    TExecutableStatementSyntax,
-    TExpressionSyntax>
-{
-    internal abstract partial class MethodExtractor
-    {
-        protected abstract partial class Analyzer
-        {
-            private sealed class SymbolMapBuilder
-            {
-                private readonly SemanticModel _semanticModel;
-                private readonly ISyntaxFactsService _service;
-                private readonly TextSpan _span;
-                private readonly Dictionary<ISymbol, List<SyntaxToken>> _symbolMap = [];
-                private readonly CancellationToken _cancellationToken;
-
-                public static Dictionary<ISymbol, List<SyntaxToken>> Build(
-                    ISyntaxFactsService service,
-                    SemanticModel semanticModel,
-                    SyntaxNode root,
-                    TextSpan span,
-                    CancellationToken cancellationToken)
-                {
-                    Contract.ThrowIfNull(semanticModel);
-                    Contract.ThrowIfNull(service);
-                    Contract.ThrowIfNull(root);
-
-                    var builder = new SymbolMapBuilder(service, semanticModel, span, cancellationToken);
-                    builder.Visit(root);
-
-                    return builder._symbolMap;
-                }
-
-                private SymbolMapBuilder(
-                    ISyntaxFactsService service,
-                    SemanticModel semanticModel,
-                    TextSpan span,
-                    CancellationToken cancellationToken)
-                {
-                    _semanticModel = semanticModel;
-                    _service = service;
-                    _span = span;
-                    _cancellationToken = cancellationToken;
-                }
-
-                private void Visit(SyntaxNode node)
-                {
```

Comment:
this entire type was unnecessary.  it was constructed to just call this method a single type.  inlined this method to the callsite.